### ISSUE TITLE
Stretch Goals

### ISSUE THREAD

**Issue by dylankmoore** [dylankmoore/TunaPiano#2]
Once our MVP for backend is complete, we can then begin working on some stretch goals:

These are examples of stretch goals that you can tackle once you have been MVP approved for the above features!

- [ ] Plan and Build the Frontend for the MVP routes
- [x] [Popular genres: Retrieve a list of genres based on the number of associated songs](https://github.com/dylankmoore/django-rest-api-assessment/blob/main/documentation/issue-tickets/Popular-genres.md)
- [x] [Related artists: Retrieve artists with similar genres](https://github.com/dylankmoore/django-rest-api-assessment/blob/main/documentation/issue-tickets/Related-artists.md)
- [x] [Search songs by genre](https://github.com/dylankmoore/django-rest-api-assessment/blob/main/documentation/issue-tickets/Search-songs-by-genre.md)
- [x] [Search artists by genre](https://github.com/dylankmoore/django-rest-api-assessment/blob/main/documentation/issue-tickets/Search-artists.md)
- [x] Search all entities by (name/title/description)
